README.md

```diff
@@ -85,11 +86,18 @@ $ kubectl -n istio-system apply -f config/samples/servicemesh_v1alpha1_istiocont
 istiocontrolplane.servicemesh.cisco.com/icp-v116x-sample created
 ```
 
-Label the namespace, where you would like to enable sidecar injection for your pods. The label should consist of the name of the deployed IstioControlPlane and the namespace where it is deployed.
+Create and label the namespace, where you would like to enable sidecar injection for your pods. The label should consist of the name of the deployed IstioControlPlane and the namespace where it is deployed.
 ```
 $ kubectl label namespace demoapp istio.io/rev=icp-v116x-sample.istio-system
 namespace/demoapp labeled
 ```
+```
+kubectl get pods -n istio-system
+NAME                                                 READY   STATUS    RESTARTS   AGE
+istio-operator-controller-manager-7875df958d-v4h7l   2/2     Running   0          17h
+istiod-icp-v116x-sample-7b499b7d-gps2b               1/1     Running   0          17h```
+```
+Note the `icp-v116x-sample` string in both outputs
```

Comment:
What do u mean by "both" here exactly? I can only see the string in "istiod-icp-v116x-sample-7b499b7d-gps2b", but not in "istio-operator-controller-manager-7875df958d-v4h7l".